rotkehlchen/tests/api/test_liquity.py

```diff
@@ -270,3 +270,21 @@ def test_staking_stats(rotkehlchen_api_server, ethereum_accounts):
     assert address_0_data['stability_pool_gains'][0]['amount'] == '1.7820064710306824'
     assert address_1_data['stability_pool_gains'][0]['amount'] == '12.294706987216218'
     assert address_1_data['stability_pool_gains'][1]['amount'] == '4240.34942308358'
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0xbB8311c7bAD518f0D8f907Cad26c5CcC85a06dC4']])
```

Comment:
I told you that another test would probably fail. have you checked it? https://github.com/rotki/rotki/actions/runs/4416177427/jobs/7740251558#step:10:1320